plugins/integration/providers/cielo24/lib/kCielo24FlowManager.php

```diff
@@ -319,4 +345,55 @@ private function integerToTime($int)
 		$stringTime .= '.' . str_pad($milliseconds, 3, 0, STR_PAD_LEFT);
 		return $stringTime;
 	}
+	
+	private function normalizeJson ($transcript)
+	{
+		$jsonElements = json_decode($transcript, true);
+		$segments = $jsonElements["segments"];
+		
+		$normalizedTokens = array ();
+		
+		foreach ($segments as $segment)
+		{
+			$sequences = $segment ['sequences'];
+			foreach ($sequences as $sequence)
+			{
+				$tokens = $sequence['tokens'];
+				foreach ($tokens as $token)
+				{
+					if (!isset ($token['type']) || 
```

Comment:
I think that this statement can be simplified to `!isset ($token['type']) || in_array($token['type'], array('word', 'punctuation'))`